examples/resources/xray_watch/resource.tf

```diff
@@ -0,0 +1,87 @@
+resource "xray_watch" "all-repos" {
+  name        = "all-repos-watch"
+  description = "Watch for all repositories, matching the filter"
+  active      = true
+  watch_resource {
+    type = "all-repos"
+    filter {
+      type  = "regex"
+      value = ".*"
+    }
+  }
+  assigned_policy {
+    name = xray_security_policy.security1.name
+    type = "security"
+  }
+  assigned_policy {
+    name = xray_license_policy.license1.name
+    type = "license"
+  }
+  watch_recipients = ["test@email.com", "test1@email.com"]
```

Comment:
Why do I feel like the new APIs work differently?

Reply:
No idea :). It works. Most of the fields are the same.